### PR TITLE
Add API for disabling video track

### PR DESCRIPTION
**Pull request by ceyusa**
This is the media player side for https://github.com/servo/servo/issues/23336

### CODE REVIEW

**Comment by ceyusa** [comment]
@ferjm r?

File: examples/player/main.rs [hunk 1]
```diff
@@ -284,15 +284,18 @@ fn main() {
 #[cfg(not(target_os = "android"))]
 fn main() {
     let args: Vec<_> = env::args().collect();
-    let (use_gl, filename) = if args.len() == 2 {
+    let (no_video, use_gl, filename) = if args.len() == 2 {
```
Comment (philn) [inline examples/player/main.rs]: May I suggest the clap (or similar crate) for CLI args parsing?

**Comment by ferjm** [comment]
#258

**Pull request merged**